ipatests/test_integration/test_multidomain_ipa.py

```diff
@@ -0,0 +1,35 @@
+from ipatests.pytest_ipa.integration import tasks
+from ipatests.test_integration.base import IntegrationTest
+
+
+class TestMultidomain(IntegrationTest):
+    num_trusted_domains = 1
+    num_clients = 1
+    num_replicas = 1
+    num_trusted_clients = 1
+    num_trusted_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
```

Comment:
We should probably have the same logic as the IntegrationTest class. Define a MultiDomainIntegrationTest class where
- the install method should perform the installation of the master/replicas/clients for IPA domain and trustedmaster/trustedreplicas/trustedclients for the trusted domain, also using install_topo (this method performs various configuration steps, for instance creation of DNS records, setting the DNS resolver etc...)
- the uninstall method should perform the uninstallation of IPA and TRUSTEDIPA domains

Then you can define a test class inheriting from this new class and add test functions inside the class.